service/policy/db/migrations/20250319000000_key_management_updates.sql

```diff
@@ -0,0 +1,302 @@
+-- +goose Up
+-- +goose StatementBegin
+DROP TRIGGER IF EXISTS trigger_update_was_mapped_namespace ON attribute_namespace_public_key_map;
+
+DROP TRIGGER IF EXISTS trigger_update_was_mapped_definition ON attribute_definition_public_key_map;
+
+DROP TRIGGER IF EXISTS trigger_update_was_mapped_value ON attribute_value_key_map;
+
+DROP TRIGGER IF EXISTS maintain_active_key ON public_keys;
+
+DROP FUNCTION IF EXISTS update_active_key;
+
+DROP VIEW IF EXISTS active_value_public_keys_view;
+
+DROP VIEW IF EXISTS active_definition_public_keys_view;
+
+DROP VIEW IF EXISTS active_namespace_public_keys_view;
+
+DROP TABLE IF EXISTS attribute_namespace_public_key_map;
+
+DROP TABLE IF EXISTS attribute_definition_public_key_map;
+
+DROP TABLE IF EXISTS attribute_value_public_key_map;
+
+DROP TABLE IF EXISTS public_keys;
+
+-- DROP FUNCTION IF EXISTS update_was_mapped;
```

Comment:
Did we still need this?

Reply:
Yeah we should still drop that function. I was probably running into issues and commented it out. Adding ti back.